test-resources/odata-service-specs/v4/API_TEST_SRV/API_TEST_SRV.edmx

```diff
@@ -255,8 +263,17 @@
         <ActionImport Name="TestActionImportMultipleParameterComplexReturnType" Action="API_TEST_SRV.TestActionImportMultipleParameterComplexReturnType">
         </ActionImport>
 
-       <ActionImport Name="TestActionImportUnsupportedEdmTypes" Action="API_TEST_SRV.TestActionImportUnsupportedEdmTypes">
-       </ActionImport>
+        <ActionImport Name="TestActionImportUnsupportedEdmTypes" Action="API_TEST_SRV.TestActionImportUnsupportedEdmTypes">
+        </ActionImport>
+
+        <ActionImport Name="TestActionImportNoParameterEntityReturnType" Action="API_TEST_SRV.TestActionImportNoParameterEntityReturnType">
+        </ActionImport>
+
+        <ActionImport Name="TestActionImportEntityParameterNoReturnType" Action="API_TEST_SRV.TestActionImportEntityParameterNoReturnType">
+        </ActionImport>
+
+        <ActionImport Name="TestActionImportNoActionDefined" Action="API_TEST_SRV.TestActionImportNoActionDefined">

```

Comment:
e.g, this action import has action `API_TEST_SRV.TestActionImportNoActionDefined`, which is not defined.